.github/workflows/test-and-release.yml

```diff
@@ -37,6 +37,7 @@ jobs:
         init-args: ["-upgrade", ""]
         validate-args: ["-json", ""]
         fmt-args: ["-check", ""]
+        test-args: ["-json"]
```

Comment:
maybe with and without option?

or move this to the `test-terraform-test` job as it won't be executed here...